src/main/java/frc/robot/commands/ClimberTranslate.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Climber;
+
+public class ClimberTranslate extends Command {
+  private Climber s_climber; 
+  private double pwr;
+  /** Creates a new ClimberTranslate. */
+  public ClimberTranslate(Climber clibmer, double pwr) {
+    this.pwr = pwr;
+    s_climber = clibmer;
+    addRequirements(s_climber);
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {}
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+
+    //put smart dashboard running command update
+    SmartDashboard.putString("Climber CMD", "Translate");
+
+    s_climber.setClimberPower(pwr);
+  }
+
+  // Called once the command ends or is interrupted.
+  @Override
+  public void end(boolean interrupted) {
+    s_climber.setClimberPower(0);
+  }
+
+  // Returns true when the command should end.
+  @Override
+  public boolean isFinished() {
+    return false;
```

Comment:
This is fine for now, but we probably want to stop climbing at some point in order not to overrun the mechanism